target/linux/ramips/mt7620/base-files/etc/board.d/01_leds

```diff
@@ -165,6 +165,13 @@ netgear,ex6130)
 	ucidef_set_led_netdev "wlan5g" "ROUTER (green)" "$boardname:green:router" "wlan0"
 	ucidef_set_led_netdev "wlan2g" "DEVICE (green)" "$boardname:green:device" "wlan1"
 	;;
+netgear,jwnr2010-v5)
+	ucidef_set_led_switch "lan1" "lan1" "$boardname:green:lan1" "switch0" "0x8"
+	ucidef_set_led_switch "lan2" "lan2" "$boardname:green:lan2" "switch0" "0x4"
+	ucidef_set_led_switch "lan3" "lan3" "$boardname:green:lan3" "switch0" "0x2"
+	ucidef_set_led_switch "lan4" "lan4" "$boardname:green:lan4" "switch0" "0x01"
```

Comment:
Why 01 here and one digit above?

Reply:
Ya, this is weird when I saw what others did. No one did it before, everyone write it as 0x01. So I just followed for the sake of consistency. @adrianschmutzler

Reply:
There is two options: 1. Consistency, which means you use two digits for _all_ addresses, i.e. 0x02, ...
2. Brevity, so you remove unnecessary leading zeros.

Reply:
got it